integration-requirements.txt

```diff
@@ -12,10 +14,8 @@ boto3==1.5.9
 paramiko==2.4.2
 cryptography==2.4.2
 
-
 # lxd backend
-# 04/03/2018: enables use of lxd 3.0
-git+https://github.com/lxc/pylxd.git@4b8ab1802f9aee4eb29cf7b119dae0aa47150779
+pylxd
```

Comment:
cloud-init has been broken multiple times by tracking tip of pylxd and we "pinned" it to a known acceptable committish to avoid that intermittent breakage. Can we just bump the commitish to more recent tip of master if we are missing something to avoid thrashing on future breaks?